web/META-INF/resources/js/services.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
Also @gxtchen is it possible for you to add ppropriate [license headers](https://github.com/mudrod/mudrod/blob/master/service/src/main/java/gov/nasa/jpl/mudrod/services/ontology/OntologyResource.java#L1-L13) for all of the new files you are adding?